.default.yml

```diff
@@ -80,8 +80,8 @@ Rails/SkipsModelValidations:
 # For this group of specs, there is no class
```

Comment:
@jmax, I confirmed that we won’t be able to go to rubocop-rspec 2.5 until we can get to rubocop 1.19, which is currently blocked by the lack of a release for the lastest changes for the Code Climate CLI. Please update the comment to that effect, i.e.:
```
# For this group of specs, there is no class.
# We can remove these exclusions once we're using rubocop-rspec >= 2.5 & rubocop >= 1.19,
# which depend on unreleased changes to the Code Climate CLI:
# https://github.com/codeclimate/codeclimate/commit/ab6ef5ce65afad5356ed3d558ac73dbc4fd4d49f

Reply:
Added.